File: tasks/2025/round1_pp/pyramid-pp.py
#!/usr/bin/env python3
# @check-accepted: examples N<=5 N<=1000 no-limits

import sys

# uncomment the two following lines if you want to read/write from files
# sys.stdin = open('input.txt')
# sys.stdout = open('output.txt', 'w')

N = int(input().strip())

L = -1
i = 1
tot = 0
while tot <= N:
    tot += i * i
    L += 1
    i += 1


print(L)

sys.stdout.close()

File: tasks/2025/round1_pp/pyramid-pp.cpp
// @check-accepted: examples N<=5 N<=1000 no-limits

#include <fstream>
#include <iostream>
#include <vector>

using namespace std;

int main() {
    // uncomment the following lines if you want to read/write from files
    // ifstream cin("input.txt");
    // ofstream cout("output.txt");

    int N;
    cin >> N;

    int L = -1, tot = 0;


    for (int i = 1; tot <= N; i++) {
        tot += i * i;
        L++;
    }


    cout << L << endl;

    return 0;
}
